test/algconvergence/symplectic_tests.jl

```diff
@@ -0,0 +1,60 @@
+module SymplecticTests
```

Comment:
You don't have to define a module here, `@safetestset` in `test/runtests.jl` already runs everything inside of a separate module automatically.

Reply:
thanks - will change